### ISSUE TITLE
binstar still mentioned in messages

### ISSUE THREAD

**Issue by pitrou** [Anaconda-Server/anaconda-build#213]
Perhaps someone should do a grep on the whole source / docs tree?

```
$ anaconda build -h
usage: anaconda build [-h]  ...

optional arguments:
  -h, --help  show this help message and exit

Commands:
  
    backlog   Run a build worker to build jobs off of a binstar build queue
    resubmit  Resubmit build
    tail      Tail the build output of build number X.Y
    list-all  list the builds for package
    list      list the builds for package
    init      Initialize Build file
    keyfiles  [Advanced] Not documented yet
    keyfile   [Advanced] Not documented yet
    queue     Inspect build queue
    results   [Advanced] Attach results to build
    save      Save build info to be triggered later
    submit    Submit a diectory or github repo for building
    trigger   Trigger a build that has been saved
    worker    DEPRECATED: use anaconda worker -h

Binstar Build command

To get started with binstar build run:

    binstar-build init
    binstar-build submit

See also:

  * [Binstar Build](http://docs.anaconda.org/examples.html#BinstarBuild)
```

**Comment by PeterDSteinberg** [comment]
I agree this is confusing and needs to be fixed.

**Comment by ijstokes** [comment]
+1

**Comment by PeterDSteinberg** [comment]
Fixed by #209 and #215
